server/monitor.go

```diff
@@ -661,12 +692,20 @@ func (s *Server) Routez(routezOpts *RoutezOptions) (*Routez, error) {
 			RTT:          r.getRTT(),
 		}
 
-		if subs && len(r.subs) > 0 {
-			ri.Subs = make([]string, 0, len(r.subs))
-			for _, sub := range r.subs {
-				ri.Subs = append(ri.Subs, string(sub.subject))
+		if len(r.subs) > 0 {
+			if opts.SubscriptionsDetail {
```

Comment:
Code very similar to what we do for connz, so maybe it can be made a function where we pass a param as to if we want detail or not, etc..